UnityProject/Assets/Scripts/Items/Headset.cs

```diff
@@ -26,6 +29,26 @@ public void ServerPerformInteraction(HandActivate interaction)
 		}
 	}
 
+	public string Examine(Vector3 worldPos = default)
+	{
+		string status = "";
+		if (isBroken)
+		{
+			status = $"<color=red>It appears to be broken.";
+		}
+		return $"{gameObject.GetComponent<ItemAttributesV2>().InitialDescription}" +
+			status;
+	}
+
+	public void OnEMP(int EMPStrength = 0)
+    {
+        if (!isBroken)
+        {
+			isBroken = true;
```

Comment:
From a gameplay perspective this could be problematic / annoying. Could you add a quick interaction w/ a screwdriver or something to repair the headset?

Oh, actually, in /tg/ it just turns the headset off, right? Perhaps do that instead, and a quick HandActivate interaction to toggle the headset's on/off state?

Reply:
Bugger, I see HandActivate is used. could move the loudspeaker toggle to AltClick for now, until we sort out a lil UI for it.

Reply:
Can just make coroutine to disable it for some amount of time